From 85dc5b50bd2951887184d54c4176faba081e230f Mon Sep 17 00:00:00 2001
From: Simon Shields <simon@lineageos.org>
Date: Sat, 20 Jan 2018 19:05:33 +1100
Subject: [PATCH 085/249] search: improve search entry of device info pages

This allows searching for the "real" name of a device (e.g.
Galaxy S5) instead of its codename.

Change-Id: Iba093c1aae46dad83feb11d3b01527e96c658ffe
---
 search.json | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/search.json b/search.json
index 0acd69a..833b71a 100644
--- a/search.json
+++ b/search.json
@@ -10,7 +10,12 @@ search: exclude
 {% unless page.search == "exclude" or page.url contains "_info.html" or page.url contains "_install.html" or page.url contains "_build.html" or page.url contains "devices.html" or page.url contains ".css" %}
 {% unless page.url contains "/devices" and page.url contains "/build" or page.url contains "/devices" and page.url contains "/install" %}
 {
+{% if page.url contains "/devices" %}
+{% assign device = site.data.devices[page.device] %}
+"title": "{{ device.codename | escape }} - {{ device.vendor | escape }} {{ device.name }}",
+{% else %}
 "title": "{{ page.title | escape }}",
+{% endif %}
 "tags": "{{ page.tags }}",
 "keywords": "{{page.keywords}}",
 "url": "{{ page.url | relative_url }}",
-- 
2.17.1

